### ISSUE TITLE
IIIF images are ghosting duplicates with any interaction in the modal

### ISSUE THREAD

**Issue by Erin-Cecele** [thegetty/quire#809]
### Before proceeding, make sure there isn’t an existing issue for this bug.

- [X] I have searched the existing issues and determined this is a new bug.

### Expected Behavior

Open an image in the lightbox modal and zoom with no duplication. 

### Actual Behavior

Per Conserving Canvas repo issue [#154](https://github.com/thegetty/conserving-canvas/issues/154)

> If you open an zooming image in the lightbox modal, any interaction (zooming in, click and drag) will leave a trail of duplicates:
> 
> <img alt="Screenshot 2023-07-25 at 2 20 25 PM" width="1595" src="https://user-images.githubusercontent.com/7796401/256041586-c0bd76e6-1b70-41e9-822f-49002dadd091.png">
> This happens both in the deployed version and locally. Critically, it seems connected to the publication being published at a sub-directory. With `url: 'https://www.getty.edu/publications/conserving-canvas/'` this issue occurs. Remove the sub-directories, making it something like `url: 'https://www.getty.edu/`, and the issue is resolved.
> 
> 
>See also issue [#150](https://github.com/thegetty/conserving-canvas/issues/150), which is also a result of the sub-directories in the `url`.

### Steps to Reproduce

N/A

### Version Numbers

N/A

### Web Browser

Firefox 

### Relevant Terminal/Shell Output

```shell
N/A
```

### Supporting Information

N/A

**Comment by Erin-Cecele** [comment]
Our developers are currently working on resolving this issue. Please stay tuned for updates.

**Comment by Erin-Cecele** [comment]
We were able to confirm that this issue is **_not_** related to subdirectories. 

We're wondering if the issue may instead be related to browser performance and repainting the canvas as we've only been able to reproduce it in larger projects with many images. 

As a next step, we have opened a ticket with Digerati: https://github.com/digirati-co-uk/canvas-panel/issues/869.

**Comment by Erin-Cecele** [comment]
Turns out that this was because I added the following to custom.css in order to address another issue I was having with the default black Atlas background showing through as a line in some images inline (https://github.com/thegetty/conserving-canvas/issues/40).
```js
:root {
  --atlas-background: transparent;
} 
```
As long as --atlas-background is anything but transparent, the ghosting doesn't happen.

It still seems like a bug to me, but since this was only an issue in this one book, I'm going to go ahead and close mark this issue as done.
